extensions/ql-vscode/package.json

```diff
@@ -634,6 +634,10 @@
         "title": "Create query",
         "icon": "$(new-file)"
       },
+      {
```

Comment:
Does this command even need to be defined in here, since it's only internal to the app?

Reply:
Good point! It probably doesn't. I'll remove it here 🔪